README.md

```diff
@@ -75,6 +75,15 @@ below for posting rules).  For instance:
 
 These metrics are sent to CloudWatch based on the rules described below.
 
+To write [high-resolution metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/publishingMetrics.html#high-resolution-metrics=), supply the `:storage_resolution` option:
+
+```elixir
+counter(
+  "http.request.count",
+  reporter_options: [storage_resolution: 1]
```

Comment:
Thanks for the PR, @reisub!  Based on [the docs](https://docs.aws.amazon.com/cli/latest/reference/cloudwatch/put-metric-data.html):

> Valid values are 1 and 60

What do you think about having `:storage_resolution` values of either `:high` or `:standard` (based on the two options listed [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/publishingMetrics.html#high-resolution-metrics) too).

```suggestion
  reporter_options: [storage_resolution: :high]
```

Reply:
Since AWS only accepts 2, I think it would be a good idea if the interface in this lib only accepts two, too.  If AWS adds more in the future (unlikely?) the lib could accept `:high` / `:standard` / or a positive integer value conforming to whatever their new rules are.

Reply:
@bmuller makes sense, pushed https://github.com/bmuller/telemetry_metrics_cloudwatch/pull/6/commits/5c5ad432c15290c8459b498303f17499a5c231ca to make it nicer.